redoc/template.html

```diff
@@ -14,6 +14,28 @@
     </script>
     <!-- End Google Analytics -->
 
+    <!-- Chameleon -->
+    <script>
+      !function(d,w){var t="SxwDe1SHyKIPk9yVrOBdgIZEJCbYyPqSsj4j9aBTi7kTE0-1KuSgl-CsEfJq8JP4SVT8gb",c="chmln",m="identify alias track clear set show on off custom help _data".split(" "),i=d.createElement("script");if(w[c]||(w[c]={}),!w[c].root){w[c].accountToken=t,w[c].location=w.location.href.toString(),w[c].now=new Date;for(var s=0;s<m.length;s++){!function(){var t=w[c][m[s]+"_a"]=[];w[c][m[s]]=function(){t.push(arguments);};}();}i.src="https://fast.trychameleon.com/messo/"+t+"/messo.min.js",i.async=!0,d.head.appendChild(i);}}(document,window);
+
+      const getOrCreateUserId = () => {
+        const allCookies = document.cookie;
+        let surveyCookie = allCookies.split('; ').find(row => row.startsWith('npsId='));
```

Comment:
@alannaidon 
```suggestion
        let surveyCookie = allCookies && allCookies.split('; ').find(row => row.startsWith('npsId='));
```
Sugiro validar o allCookies, antes fazer o split. Não sei se é o caso, mas se ele vier com os valores `null` ou `undefined` pode quebrar a execução. 

Com essa validação, caso ocorra essa situação de vir `null` ou `undefined` o valor da variável será `undefined`.

Veja se faz sentido ai para você.